packages/backend/test/pg-reset-db-setup.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-console */
+import '../src/config/orm'
+
+import { readdirSync } from 'fs'
+import knex, { Knex } from 'knex'
+import { join } from 'path'
+import { afterEach, beforeEach } from 'vitest'
+
+import config from '../knexfile'
+
+const client = knex(config as Knex.Config)
+
+beforeEach(async () => {
+  // manually running seeds for the same reasons
```

Comment:
nit: do we actually need seed files? Not all tests need a user, for example. We can force tests to pre-create their own test-specific seed data in `beforeEach`

Reply:
i dont see a mutation to DB that doesnt require at least a user. :/

Reply:
maybe only login for new user